server/crawler/crawler.go

```diff
@@ -249,15 +246,14 @@ func ReadPackage(
 		0, // We don't know (or care) what the star count is
 	)
 	packageRepositoryLocator := kurtosisPackageMetadata.GetLocator()
-	kurtosisPackageContent, ok, err := extractKurtosisPackageContent(ctx, githubClient, kurtosisPackageMetadata)
-	if !ok {
-		parsingError := stacktrace.NewError(fmt.Sprintf("Kurtosis package repository content '%s' could not be retrieved as it was invalid.", packageRepositoryLocator), err)
-		logrus.Warn(parsingError)
-		return nil, parsingError
-	}
+	kurtosisPackageContent, packageFound, err := extractKurtosisPackageContent(ctx, githubClient, kurtosisPackageMetadata)
 	if err != nil {
-		return nil, stacktrace.Propagate(err, "an unexpected error occurred retrieving content for Kurtosis package repository '%s'",
-			packageRepositoryLocator)
+		return nil, stacktrace.Propagate(err, "An error occurred extracting content for Kurtosis package repository '%s'", packageRepositoryLocator)
+	}
+	if !packageFound {
+		noPackageErr := stacktrace.NewError("No Kurtosis package found. Ensure that a package exists at '%v' with valid '%v' and '%v' files.", packageRepositoryLocator, kurtosisYamlFileName, starlarkMainDotStarFileName)
+		logrus.Warn(noPackageErr)
```

Comment:
I just thought of this: we should make sure that it's not possible for anyone to call this endpoint and submit data that we then print into the logs. it's a security problem.

Reply:
if you parse the input and clean it' then it's ok, else omit that information from the logs

Reply:
good point. should I not log the `packageRepositoryLocator` in this cas?

Reply:
not if it's not cleaned up. if I can inject malicious code through that input we should not log it